Library/Formula/riak.rb

```diff
@@ -8,6 +6,7 @@ class Riak < Formula
 
   depends_on :macos => :mountain_lion
   depends_on :arch => :x86_64
+  depends_on "openssl" => :recommended
```

Comment:
You can remove this sadly. The binary ignores it so we're stuck unless we build from source.